lib/Driver/Driver.cpp

```diff
@@ -112,6 +112,7 @@ void Driver::parseDriverKind(ArrayRef<const char *> Args) {
           .Case("swift-api-extract", DriverKind::APIExtract)
           .Case("swift-api-digester", DriverKind::APIDigester)
           .Case("swift-cache-tool", DriverKind::CacheTool)
+          .Case("swift-parse-test", DriverKind::ParseTest)
```

Comment:
Do you think we should extend this tool to do more parser tests? If it should just stay a performance measuring tool, I would name it `swift-parser-performance-test`. 

Also, I just want to note that swift-syntax has `swift-parser-test` (with an `r`). I don’t know if it’s good that this is spelled without an `r` (means we don’t have two utilities with the same name) or bad (inconsistency).